src/components/Settings/People/DeleteConfirmationDialog.js

```diff
@@ -67,6 +95,34 @@ const DeleteConfirmationDialog = ({
           </Button>
         </DialogActions>
       )}
+      {!isDeletingAccount && showConfirmationInput && (
+        <>
+          <TextField
```

Comment:
This component needs to be inside the `<DialogContent>`

Reply:
here I had a problem when I put the text field element inside DialogContent it shrinks and sticks to the left , also if its inside DialogActions , or outside of DialogContent in general it sticks to both sides of the dialog which doesn't look good , I know using inline css is not a best practice but I didn't know whether to create a new css file and if so where should I put it .

Reply:
also made sense to me to put it inside DialogActions because its kinda serves a role of a button , you know what I mean ? anyway its ok Im here to learn :)

Reply:
I see!! Take a look on material ui documentation under form dialog [here](https://v4.mui.com/components/dialogs/). The text field it's inside the dialog content. Use that as a reference. Also you can take a look on the text field props. 
You can add the fullWidth  prop to deal with your problem!

Reply:
oh got it , I have committed the changes @RodriSanchez1